site-src/enhancements.md

```diff
@@ -0,0 +1,73 @@
+# Network Policy Enhancement Proposal (NPEP)
+
+Network Policy Enhancement Proposals (NPEP) serve a similar purpose to the [KEP][kep]
+process for the main Kubernetes project:
+
+1. Ensure that changes to the API follow a known process and discussion
+   in the OSS community.
+2. Make changes and proposals discoverable (current and future).
+3. Document design ideas, tradeoffs, decisions that were made for
+   historical reference.
+
+## Process
+
+### 1. Discuss with the community
+Before creating a NPEP, share your high level idea with the community. Check
+[Community, discussion, contribution, and support](/#community-discussion-contribution-and-support)
+for the suitable communication option.
+
+### 2. Create an Issue
+[Create a NPEP issue](https://github.com/kubernetes-sigs/network-policy-api/issues/new?assignees=&labels=kind%2Ffeature&template=enhancement.md) 
+in the repo describing your change.
+At this point, you should copy the outcome of any other conversations or documents
+into this Issue.
+
+### 3. Create a first PR for your NPEP
+NPEP process is supposed to be iterative, adding more details with every iteration.
+To start this process, create a PR adding `npep-<issue number>.md` file in [npep folder](https://github.com/kubernetes-sigs/network-policy-api/tree/master/npep)
+using [template NPEP](https://github.com/kubernetes-sigs/network-policy-api/blob/master/npep/npep-95.md). Further we describe a number of 
+suggested iterations for an NPEP to make sure every version has a specific goal and limited scope.
+
+### 4. Agree on the Goals
+Although it can be tempting to start writing out all the details of your
+proposal, it's important to first ensure we all agree on the goals. The first
+version of your NPEP should leave out any implementation details, 
+focusing primarily on "Goals" and "Non-Goals".
+
+### 5. Document Implementation Details
+Now that everyone agrees on the goals, it is time to start writing out your
+proposed implementation details. These implementation details should be very
+thorough, including the proposed API spec, and covering any relevant edge cases.
+Note that it may be helpful to use a shared doc for part of this phase to enable
+faster iteration on potential designs.
+
+It is likely that throughout this process, you will discuss a variety of
+alternatives. Be sure to document all of these in the NPEP, and why we decided
+against them.
+
+### 6. Make API changes
```

Comment:
Should we note here that this step shouldn't alter the NPEP itself?

Reply:
good point! now with the statuses, status change will be required, but it is usually also useful to add links about how this NPEP was implemented, I added this to the description